woocommerce/woocommerce
# Product Editor: Modal block editor (description) small viewport (mobile) support

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





The experience of editing a description in the full editor in mobile viewport was pretty broken. This takes some steps to fix them, mostly by showing the sidebars in a fullscreen mode.

Closes #47152.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new product editor at /wp-admin/admin.php?page=wc-settings&tab=advanced&section=features
2. Go to Products > Add
3. Simulate a mobile device in portrait mode through developer tools
4. Go to Description and press "Full editor"
5. Write any description
6. Open the "Document overview" sidebar, verify it fills the entire dialog
7. Close it and open the Settings sidebar
8. Verify it fills the entire dialog
9. Click on "+" to insert a block
10. Select a block
11. Verify it gets added and the inserter is closed



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
